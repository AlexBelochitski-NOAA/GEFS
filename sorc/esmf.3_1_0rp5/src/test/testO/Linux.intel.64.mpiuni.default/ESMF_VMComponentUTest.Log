20190610 112235.364308 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112235.364602 INFO     PET0  Beginning Test, file ESMF_VMComponentUTest.F90, line         219
20190610 112235.364619 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112235.366187 INFO     PET0  PASS  Component Create/SetServices/Destroy Test, ESMF_VMComponentUTest.F90, line         252
20190610 112235.366204 INFO     PET0  Number of failed tests:           0
20190610 112235.366210 INFO     PET0  Ending Test, file ESMF_VMComponentUTest.F90, line         289
