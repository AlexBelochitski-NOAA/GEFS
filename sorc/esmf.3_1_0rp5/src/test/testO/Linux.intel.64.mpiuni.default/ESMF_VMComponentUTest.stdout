  Beginning Test, file ESMF_VMComponentUTest.F90, line         219
  NUMBER_OF_PROCESSORS           1
  PASS  Component Create/SetServices/Destroy Test, ESMF_VMComponentUTest.F90, li
 ne         252
  Number of failed tests:           0
  Ending Test, file ESMF_VMComponentUTest.F90, line         289
--- ESMCI::VM::print() start ---
ESMCI::VMIdPrint:
vmID located at: 0xb42620
  vmKey=0x80000000
  localID: 0
--- VMK::print() start ---
vm located at: 0x1a19060
npets = 1, mypet=0
  pth_mutex =		 0x1a40860
  pth_finish_count =	 (nil)
MPI_Group_size: 1
MPI_Comm_size: 1
MPI_Comm_rank in local MPI communicator: 0
MPI_Comm_rank in MPI_COMM_WORLD: 0
MPI thread level support: 0
mpi_mutex_flag: 1
mpionly: 0
nothreadsflag: 1
  lpid[0]=0, tid[0]=0, vas[0]=0, ncpet[0]=1, cid[0][0]=0
ncores = 1
  cpuid[0]=0, ssiid[0]=0
--- VMK::print() end ---
--- ESMCI::VM::print() end ---
